docs/langref.md

```diff
@@ -274,6 +284,51 @@ This has the effect that, e.g., `Π T: * → T → T` has the expected binding l
 Otherwise, `→` would be consumed by the domain: `Π T:` (`* →` (`T → T`)) ↯.
 A similar situation occurs for a `.lam` declaration.
 
+### Functions \& Types
+
+The following table summarizes the different tokens used for functions declarations, expressions, and types:
+
+| Declaration | Expression     | Type                    |
+|-------------|----------------|-------------------------|
+| `.lam`      | `.lm` <br> `λ` | `Π` <br> <tt>\|~\|</tt> |
+| `.con`      | `.cn`          | `.Cn`                   |
+| `.fun`      | `.fn`          | `.Fn`                   |
+
+#### Declarations
+
+The following function *declarations* are all equivalent:
+```
+.lam f(T: *)((x y: T), return: T -> ⊥) -> ⊥ = return x;
+.con f(T: *)((x y: T), return: .Cn T)       = return x;
+.fun f(T: *) (x y: T)                       = return x;
+```
+
+#### Expressions
+
+The following function *expressions* are all equivalent.
+What is more, since they are bound by a *let declaration*, they have the exact same effect as the function *declarations* above:
+```
+.let f = .lm (T: *)((x y: T), return: T -> ⊥) -> ⊥ = return x;
+.let f = .cn (T: *)((x y: T), return: .Cn T)       = return x;
+.let f = .fn (T: *) (x y: T)                       = return x;
+```
+
+#### Applications
+
+The following expressions for applying `f` are also equivalent:
+```
+f .Nat ((23, 42),.cn res: .Nat = use(res))
+.ret res = f .Nat : (23, 42); use(res)
+```
+#### Function Types
+
+Finally, the following function types are all equivalent and denote the type of `f` above.
+```
+ Π [T:*][T, T][T -> ⊥] -> ⊥
+.Cn[T:*][T, T][.Cn T]
```

Comment:
Shouldn't it be 
```
 Π [T:*][[T, T][T -> ⊥]] -> ⊥
.Cn[T:*][[T, T][.Cn T]]
```

If not spelled out explicitly, the text should make clear that `.Cn A` is syntactic sugar for `A -> .Bot`.